test/regression/external/user-registration.spec.js

```diff
@@ -0,0 +1,141 @@
+  });
+
+  it('navigates to the success page if the email address is valid', () => {
+    $('#email').setValue(EMAIL_ADDRESS);
+    getButton('Continue').click();
+    const uri = `/success?${qs.encode({ email: EMAIL_ADDRESS })}`;
+    expect(browser).toHaveUrlContaining(uri);
+    expect(getPageTitle()).toHaveText('Confirm your email address');
+  });
+
+  it('shows confirmation text including the email address', () => {
+    expect(getByTestId('success-text')).toHaveText(`We have sent a link to ${EMAIL_ADDRESS}`);
+  });
+
+  it('clicks the link in the confirmation email', () => {
+    const link = getPersonalisation(EMAIL_ADDRESS, 'link');
+    browser.url(link);
+    expect(browser).toHaveUrlContaining('/create-password?');
+    expect(getPageTitle()).toHaveText('Create a password');
+  });
+
+  it('shows a validation error message if no passwords are entered', () => {
+    $('#password').setValue('');
+    $('#confirm-password').setValue('');
+    getButton('Change password').click();
+    expectPasswordValidationToBe(false, false, false);
+  });
+
+  it('shows a validation error message if passwords are too short', () => {
+    $('#password').setValue('short');
+    $('#confirm-password').setValue('short');
+    getButton('Change password').click();
+    expectPasswordValidationToBe(false, false, false);
+  });
+
+  it('shows a validation error message if passwords have correct length only', () => {
+    $('#password').setValue('12345678');
+    $('#confirm-password').setValue('12345678');
+    getButton('Change password').click();
+    expectPasswordValidationToBe(true, false, false);
+  });
+
+  it('shows a validation error message if passwords have symbol only', () => {
+    $('#password').setValue('123$');
+    $('#confirm-password').setValue('123$');
+    getButton('Change password').click();
+    expectPasswordValidationToBe(false, true, false);
+  });
+
+  it('shows a validation error message if passwords have capital only', () => {
+    $('#password').setValue('123A');
+    $('#confirm-password').setValue('123A');
+    getButton('Change password').click();
+    expectPasswordValidationToBe(false, false, true);
+  });
+
+  it('shows a validation error message if passwords do not match', () => {
+    $('#password').setValue('A12345678$');
+    $('#confirm-password').setValue('B12345678$');
+    getButton('Change password').click();
+    expect(getValidationSummaryMessage()).toHaveText('Re-enter your new password');
+  });
+
+  it('sets the passwords and signs in if valid and matching', () => {
+    $('#password').setValue('A12345678$');
+    $('#confirm-password').setValue('A12345678$');
+    getButton('Change password').click();
+    expect(browser).toHaveUrlContaining('/add-licences');
+    expect(getPageTitle()).toHaveText('Add your licences to the service');
+  });
+
+  /**
+   * @todo flow where user enters email twice
+   */
+
+  after(async () => {
+    await tearDown();
```

Comment:
Instead of calling this tearDown() function which in turns calls a REST endpoint, you may find it easier/more maintainable to use the built-in Wdio event listeners like [this one here](https://github.com/DEFRA/water-abstraction-ui/blob/89701a81376e75ed17dded98a660e848d3664426/wdio.conf.js#L270) alongside the setup routine that happens [here](https://github.com/DEFRA/water-abstraction-ui/blob/89701a81376e75ed17dded98a660e848d3664426/wdio.conf.js#L167).